ecosystem/python/sdk/examples/transaction-batching.py

```diff
@@ -2,26 +2,203 @@
+        )
+
+    async def increase_transaction_count(self, number: int):
+        if number <= 0:
+            return
+
+        async with self._lock:
+            self._remaining_transactions += number
+            self._waiting_for_more.set()
+            self._complete.clear()
+
+    async def wait(self):
+        await self._complete.wait()
+
+
+class WorkerContainer:
+    _conn: Connection
+    _process: Process
+
+    def __init__(self, node_url: str, account: Account, recipient: AccountAddress):
+        (self._conn, conn) = Pipe()
+        self._process = Process(
+            target=Worker.run, args=(conn, node_url, account, recipient)
+        )
+
+    def get(self) -> typing.Any:
+        self._conn.recv()
+
+    def join(self):
+        self._process.join()
+
+    def put(self, value: typing.Any):
+        self._conn.send(value)
+
+    def start(self):
+        self._process.start()
+
+
+class Worker:
+    _conn: Connection
+    _rest_client: RestClient
+    _account: Account
+    _recipient: AccountAddress
+    _txn_generator: TransactionGenerator
+    _txn_worker: TransactionWorker
+
+    def __init__(
+        self,
+        conn: Connection,
+        node_url: str,
+        account: Account,
+        recipient: AccountAddress,
+    ):
+        self._conn = conn
+        self._rest_client = RestClient(node_url)
+        self._account = account
+        self._recipient = recipient
+        self._txn_generator = TransactionGenerator(self._rest_client, self._recipient)
+        self._txn_worker = TransactionWorker(
+            self._account, self._rest_client, self._txn_generator.next_transaction
+        )
+
+    def run(queue: Pipe, node_url: str, account: Account, recipient: AccountAddress):
+        worker = Worker(queue, node_url, account, recipient)
+        asyncio.run(worker.arun())
+
+    async def arun(self):
+        print(f"hello from {self._account.address()}", flush=True)
+        try:
+            self._txn_worker.start()
+
+            self._conn.send(True)
+            num_txns = self._conn.recv()
+
+            await self._txn_generator.increase_transaction_count(num_txns)
+
+            print(f"Increase txns from {self._account.address()}", flush=True)
+            self._conn.send(True)
+            self._conn.recv()
+
+            txn_hashes = []
+            while num_txns != 0:
+                num_txns -= 1
+                (
+                    sequence_number,
+                    txn_hash,
+                    exception,
+                ) = await self._txn_worker.next_processed_transaction()
+                if exception:
+                    print(
+                        f"Account {self._txn_worker.account()}, transaction {sequence_number} submission failed: {exception}"
+                    )
+                else:
+                    txn_hashes.append(txn_hash)
+
+            print(f"Submit txns from {self._account.address()}", flush=True)
```

Comment:
I would change this to `Submitted txns` at this point they're already submitted.